spring-cloud-azure-autoconfigure/src/main/resources/META-INF/project.properties

```diff
@@ -0,0 +1 @@
+project.version=@project.version@
```

Comment:
Why this properties is inside META-INF?

Reply:
From convention here, telemetry will take this version as property.